src/main/kotlin/app/services/impl/HttpKeyService.kt

```diff
@@ -28,17 +28,27 @@ class HttpKeyService(private val httpClientProvider: HttpClientProvider) : KeySe
 
     companion object {
         val logger: Logger = LoggerFactory.getLogger(HttpKeyService::class.toString())
-        const val maxAttempts = 10
-        const val backoffMillis = 1000L
+
+        // Will retry at 1s, 2s, 4s, 8s, 16s then give up (after a total of 31 secs)
+        const val maxAttempts = 5
```

Comment:
DW-2680 has a suggested 10 retries in the AC, should we mirror that here or is there a reason to have less?

Reply:
We've talked about it and gone for a longer total time, so will update the ticket.